language-server/modules/langserver-core/src/test/resources/codeaction/type-cast/source/type_cast_in_obj_fields.bal

```diff
@@ -1,10 +1,8 @@
+float f1 = 10.2;
```

Comment:
Please get these changes reviewed.

Reply:
@TharushiJay, can you review LS changes.